### PR TITLE
Push tomviz forward

### PR DESCRIPTION
**Pull request by cjh1**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by cjh1** [comment]
@conda-forge-admin, please rerender

**Comment by cjh1** [comment]
@cryos This pushes things forward.

**Comment by cjh1** [comment]
@conda-forge-admin, please rerender

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Home: https://github.com/OpenChemistry/tomviz
 
 Package license: BSD-3-Clause
 
-Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/tomviz-feedstock/blob/master/LICENSE.txt)
+Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/tomviz-feedstock-1-feedstock/blob/master/LICENSE.txt)
```
Comment (mrakitin) [inline README.md:8]: The links here do not look right. Any idea why it's done that way?

**Review by mrakitin — COMMENTED** [review]
Minor concern about the incorrect updates to `README.md`.

File: README.md (hunk 1 above)
Comment (cjh1) [reply README.md:8]: Strange, I am guess the re-render did it?

Comment (cjh1) [reply README.md:8]: That is the name of my fork.

Comment (cjh1) [reply README.md]: I fixed it up ...

Comment (mrakitin) [reply README.md]: Thanks! I saw a few more similar issues there. I think we should fix it the way conda-smithy does not revert it back.

Comment (cjh1) [reply README.md]: Ok, I will remove my commit

**Comment by cjh1** [comment]
@mrakitin Are you happy for me to merge, I reverted my fix?

**Comment by cjh1** [comment]
I am going to go ahead and merge this

**Comment by mrakitin** [comment]
@cjh1, sounds good. I was off last week, sorry for the late reply.

**Pull request merged**